firebase_admin/dynamic_links.py

```diff
@@ -151,16 +117,28 @@ class EventStats(object):
 
     def __init__(self, platform, event, count):
         """Create new instance of EventStats(platform, event, count)"""
-        self.platform = platform
-        self.event = event
-        self.count = count
+        if isinstance(platform, six.string_types) and platform in self._platforms.keys():
```

Comment:
These checks are doing way too much IMO. I think if you just do something like the following, that should be enough (we don't expect developers to call these directly):

```
if platform not in self._platform.values():
    raise ValueError('Invalid platform value....')
# Same for event
```

Reply:
ok. I was thinking that if a developer is using these as mocks, e.g. in tests, they might instantiate this class (which is public) in which case. (but I guess if that's what they're doing, they're fine.)